README.md

```diff
@@ -149,6 +149,49 @@ juju run postgresql-k8s/leader restore backup-id=YYYY-MM-DDTHH:MM:SSZ --wait 5m
 ```
 More details found [here](https://charmhub.io/postgresql-k8s/docs/h-restore-backup).
 
+## Scaling
+The Superset charm consists of 3 services (server, worker and beat), the server and worker services are independently scalable.
+- Server (`app-gunicorn`): Web server.
+- Worker (`worker`): for execution of jobs.
+- Beat (`beat`): for job scheduling.
+Communication between these services is via Redis which acts as a message broker. Due to the nature of the beat scheduler, it should not be scaled as this could cause some conflicing schedules. The remaining services can be scaled according to your needs.
```

Comment:
is there a way we can prevent that?

Reply:
From what I can see nothing very tested. There is [this thread](https://chat.charmhub.io/charmhub/pl/57jc9k7aw7gtpfngg4ezaq9kpw) which gives some ideas but they seem quite hacky to me. There is a request for this capability here: https://bugs.launchpad.net/juju/+bug/1901964